schema.sql

```diff
@@ -38,4 +38,28 @@ ALTER TABLE animals
   ADD owner_id INT,
   ADD CONSTRAINT fk_owner_id
   FOREIGN KEY (owner_id)
-  REFERENCES owners(id);
\ No newline at end of file
+  REFERENCES owners(id);
+
+CREATE TABLE vets(
+   id SERIAL PRIMARY KEY NOT NULL,
+   name VARCHAR,
+   age INT,
+   date_of_graduation date
+);
+
+CREATE TABLE specializations (
+   id SERIAL PRIMARY KEY NOT NULL,
+	species_id INT NOT NULL,
+	vets_id INT NOT NULL,
```

Comment:
[OPTIONAL]
- What do you think about writing the `vets_id` foreign key in a singular term? What I mean is it should be `vet_id` instead of `vets_id`. This will help to be more in line with the convention as the column is meant to reference just one vet. 😀